WordPress/src/main/java/org/wordpress/android/support/SupportWebViewActivity.kt

```diff
@@ -0,0 +1,75 @@
+package org.wordpress.android.support
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.Menu
+import androidx.activity.result.contract.ActivityResultContract
+import androidx.webkit.WebViewAssetLoader
+import androidx.webkit.WebViewAssetLoader.AssetsPathHandler
+import androidx.webkit.WebViewAssetLoader.DEFAULT_DOMAIN
+import androidx.webkit.WebViewAssetLoader.ResourcesPathHandler
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.support.SupportWebViewActivity.OpenChatWidget.ChatDetails
+import org.wordpress.android.ui.WPWebViewActivity
+
+class SupportWebViewActivity : WPWebViewActivity(), SupportWebViewClient.SupportWebViewClientListener {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        toggleNavbarVisibility(false)
+
+        val assetLoader = WebViewAssetLoader.Builder()
+            .addPathHandler("/assets/", AssetsPathHandler(this))
+            .addPathHandler("/res/", ResourcesPathHandler(this))
+            .build()
+
+        configureWebView()
+        mWebView.webViewClient = SupportWebViewClient(this, assetLoader)
+        mWebView.loadUrl("https://$DEFAULT_DOMAIN/assets/support_chat_widget.html")
```

Comment:
We can get the URL from `URL_TO_LOAD` extra here.

Reply:
See above